### DISCUSSION TITLE
SaveChangesWithoutTriggersAsync()

### DISCUSSION THREAD

**Discussion by nahropro** [koenbeuk/EntityFrameworkCore.Triggered#181] [General]
I have an after save trigger, in that trigger I update some entity and for avoid infinite loop I make a call to SaveChangesWithoutTriggersAsync(), but it throws this exception 'System.InvalidOperationException: 'A triggerSession has already been created''!

**Comment by adrnchk** [comment]
I have the exact same problem, please help
